app/mailers/candidate_mailer.rb

```diff
@@ -282,7 +282,7 @@ def changed_offer(application_choice)
     @current_course_option = @application_choice.current_course_option
     @is_awaiting_decision = application_choice.self_and_siblings.decision_pending.any?
     @offers = @application_choice.self_and_siblings.select(&:offer?).map do |offer|
```

Comment:
Is this block argument a bit confusing now that we have an `Offer` model?

Reply:
isn't this just looking at the status though?

Reply:
I mean, it could be `choice`?

Reply:
ah lol your talking about naming, yea will change

Reply:
i find `choice` to be confusing considering we use `application_choice` across the app, at least on the provendor side and choice could easily mean course choice! If anything, it's an `application` and it was probably named `ApplicationChoice` so it doesn't clash with `Rails::Application` (this is not something to address as part of this PR, I just really do not like `choice` 😛 )

Reply:
agree, although only reason to go with `choice` here was because we have an `application_choice` already in that method